### PR TITLE
Add SMS country picker for default code

### PR DESCRIPTION
**Pull request by sethvargo**
**Release Note**

```release-note
Allow realms to select a default country code for SMS messages
```

screenshot-20200923-180315

### CODE REVIEW

**Comment by sethvargo** [comment]
Fixes GH-649

**Comment by sethvargo** [comment]
/hold 

while it looks nice, it doesn't actually add the prefix...

**Comment by mikehelmick** [comment]
it does look nice!

**Comment by sethvargo** [comment]
/unhold

I forgot this is javascript submitted.

File: cmd/server/assets/home.html [hunk 1]
```diff
@@ -260,6 +260,18 @@ <h1>Create verification code</h1>
       $uuidConfirm = $('#uuid-confirm');
         $uuid = $('#uuid');
 
+      {{if $hasSMSConfig}}
+      // Initialize pretty phone
+      let phone = document.querySelector('#phone');
+      let iti = window.intlTelInput(phone, {
+        nationalMode: true,
+        {{- if $currentRealm.SMSCountry }}
```
Comment (whaught) [inline cmd/server/assets/home.html:268]: ...I should totally be making more use of `{{-` for nicer layout

**Review by whaught — APPROVED** [review]
/lgtm

File: cmd/server/assets/home.html (hunk 1 above)
Comment (sethvargo) [reply cmd/server/assets/home.html:268]: I really try to avoid it unless specifically required. I don't particularly care in HTML, but js is... finicky.

**Pull request merged**